scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/StepSimulationTest.java

```diff
@@ -0,0 +1,64 @@
+package gov.nasa.jpl.aerie.scheduler;
+
+import gov.nasa.jpl.aerie.merlin.driver.SerializedActivity;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import org.apache.commons.lang3.tuple.Triple;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.SECOND;
+
+public class StepSimulationTest {
+
+  @Test
+  public void resetEveryTime() {
+    manyacts(true);
+  }
+  @Test
+  public void dontreset() {
+    manyacts(false);
+  }
+
+  public void manyacts(boolean reset){
+    var cur = Duration.ZERO;
+    final var step = Duration.of(3,SECOND);
+    // ten days
+    final var durSim = Duration.of(864000, SECOND);
+
+    final var acts = new ArrayList<Triple<Duration, SerializedActivity, String>>();
+    for(; cur.shorterThan(durSim); cur = cur.plus(step) ){
+      var act = Triple.of(
+          cur,
+          new SerializedActivity("BasicActivity", Map.of()), String.valueOf(cur));
```

Comment:
Two of the components of this "triple" are on one line, making it hard to parse its components visually.
```suggestion
      var act = Triple.of(
          cur,
          new SerializedActivity("BasicActivity", Map.of()),
          String.valueOf(cur));
```